### PR TITLE
Add new url property usage description. 

### PR DESCRIPTION
**Pull request by mendirattanishant**
In PR #461 we introduced new property `url` in `MQTTWebsocketTransport` to accept custom pre signed url for connection.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,18 @@ Publish a message to a topic:
 }];
 ```
 
+If you already have a self signed URL from broker like AWS IoT endpoint, use the url property of MQTTWebsocketTransport:
```
[the remark this answers is not in the document]
Comment (mendirattanishant) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -73,6 +73,18 @@ Publish a message to a topic:
...
+```objective-c
+MQTTWebsocketTransport *transport = [[MQTTWebsocketTransport alloc] init];
+transport.url = @"wss://aws.iot-amazonaws.com/mqtt?expiry='2018-05-01T23:12:32.950Z'"
+
+MQTTSession *session = [[MQTTSession alloc] init];
+session.transport = transport;
+[session connectWithConnectHandler:^(NSError *error) {
+// Do some work
```
[the remark this answers is not in the document]
Comment (mendirattanishant) [reply README.md]: Oops. Sorry missed this one.

**Comment by mendirattanishant** [comment]
@jcavar  took care of comments.

**Comment by jcavar** [comment]
Nice! Thank you @mendirattanishant!

**Pull request merged**